src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -176,6 +176,7 @@ private-f net.minecraft.client.renderer.model.ModelBakery field_217853_J # field
 protected net.minecraft.client.renderer.model.ModelBakery func_177594_c(Lnet/minecraft/util/ResourceLocation;)Lnet/minecraft/client/renderer/model/BlockModel; # loadModel
 private-f net.minecraft.client.renderer.tileentity.PistonTileEntityRenderer field_178462_c # blockRenderer - it's static so we need to un-finalize in case this class loads to early.
 public net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher field_147557_n # fontRenderer - needed for rendering text in TESR items before entering world
+default net.minecraft.client.settings.KeyBinding field_74513_e # pressed
```

Comment:
Shouldn't this be public or protected or something given it is in a class and not an interface (if default even works in ATs in interfaces, no idea)

Reply:
I made it default, because the field should not be used my modders.

Reply:
The access you want is protected, so that the subclass ToggleableKeyBinding can access it, but others cant.

Reply:
I know what protected does. I specifically did _not_ make it protected because (like I said above) mods should not mess with this field, imho.

Reply:
Works for me.